src/autoIncrement.ts

```diff
@@ -92,20 +94,31 @@ export function AutoIncrementID(schema: mongoose.Schema<any>, options: AutoIncre
       return;
     }
 
+    const modelName = opt.model || (this.constructor as any).modelName;
+    logger.info('modelName "%s"', modelName);
+
     if (!model) {
       logger.info('Creating idtracker model named "%s"', opt.trackerModelName);
       model = this.db.model(opt.trackerModelName, IDSchema, opt.trackerCollection);
+      const counter: AutoIncrementIDTrackerSpec = await model.findOne({ model: modelName, field: opt.field }).lean().exec();
+      if (!counter) {
+        await model.create({
+          model: modelName,
+          field: opt.field,
+          count: opt.startAt > 0 ? opt.startAt - opt.incrementBy : 0
+        } as AutoIncrementIDTrackerSpec);
+      }
```

Comment:
could you add an test for this? (that runs it once, checks, runs again to expect an error ("Duplicate")?

Reply:
its working fine. no `Duplicate` error.

Reply:
> its working fine. no Duplicate error.

i meant to ask if you could add an test for this new (separate) "query"

Reply:
as a note, i will add one later when this is merged